packages/los-avdelingsleder/src/grupper/components/GruppeSaksbehandlere.tsx

```diff
@@ -0,0 +1,114 @@
+
+interface Props {
+  saksbehandlerGruppe: SaksbehandlerGruppe;
+  avdelingensSaksbehandlere: SaksbehandlerAvdeling[];
+  lagreValgtSaksbehandlar: (brukerIdent: string, gruppeId: number, isChecked: boolean) => void;
+  endreGruppenavn: (gruppeId: number, gruppeNavn: string) => void;
+}
+
+const GruppeSaksbehandlere: FunctionComponent<Props> = ({
+  saksbehandlerGruppe,
+  avdelingensSaksbehandlere,
+  lagreValgtSaksbehandlar,
+  endreGruppenavn,
+}) => {
+  const intl = useIntl();
+
+  const formMethods = useForm({
+    defaultValues: {
+      navn: saksbehandlerGruppe.gruppeNavn,
+    },
+  });
+
+  const sorterteGrupperteSaksbehandlere = sortGrupperteSaksbehandlere(saksbehandlerGruppe.saksbehandlere);
+  const sorterteSaksbehandlereForAvdeling = sortAvdelingensSaksbehandlere(avdelingensSaksbehandlere);
+
+  return (
+    <Form formMethods={formMethods}>
+      <VStack gap="5">
+        <InputField
+          name="navn"
+          label={intl.formatMessage({ id: 'GruppeSaksbehandlere.Navn' })}
+          validate={[required, minLength3, maxLength100, hasValidName]}
+          onChange={(navn: string) => endreGruppenavn(saksbehandlerGruppe.gruppeId, navn)}
+          className={styles.navn}
+        />
+        <VStack gap="1">
+          <Label size="small">
+            <FormattedMessage id="GruppeSaksbehandlere.ValgteSaksbehandlere" />
+          </Label>
+          <VStack gap="2">
+            {sorterteGrupperteSaksbehandlere.length === 0 && (
+              <BodyShort size="small">
+                <FormattedMessage id="GruppeSaksbehandlere.Ingen" />
+              </BodyShort>
+            )}
+            {sorterteGrupperteSaksbehandlere?.map(saksbehandler => (
+              <HStack gap="2" align="center">
+                <div>{`${saksbehandler.navn} (${saksbehandler.brukerIdent})`}</div>
+                <div>
+                  <XMarkIcon
+                    onMouseDown={() =>
+                      lagreValgtSaksbehandlar(
+                        saksbehandler.brukerIdent,
+                        saksbehandlerGruppe.gruppeId,
+                        false,
+                      )
+                    }
+                    onKeyDown={() =>
+                      lagreValgtSaksbehandlar(
+                        saksbehandler.brukerIdent,
+                        saksbehandlerGruppe.gruppeId,
+                        false,
+                      )
+                    }
+                  />
+                </div>
+              </HStack>
+            ))}
+          </VStack>
+        </VStack>
+        {/* eslint-disable-next-line react/jsx-pascal-case */}
+        <UNSAFE_Combobox // eslint-disable-line camelcase
+          label={intl.formatMessage({ id: 'GruppeSaksbehandlere.VelgSaksbehandlere' })}
+          size="small"
+          options={sorterteSaksbehandlereForAvdeling.map(
+            saksbehandler => `${saksbehandler.navn} (${saksbehandler.brukerIdent})`,
+          )}
+          onToggleSelected={option => {
```

Comment:
Prøver å legge til flere på en gang og har noen observasjoner:
1. autocomplete er bokstav for bokstav - mulig med litt bedre matching?
2. finner riktig fra nedtrekk, trykker enter -> liste blir oppdatert ok men gruppen kollapser. Vil ha expanded visning fortsatt
3. det er stort antall saksbehandlere i prod, behøver kanskje ikke vise noen før avdelingsleder har begynt med input her. Greit å også filtrere ut allerede valgte saksbehandlere.